### PR TITLE
Added the command to install babel-preset for babel-loader

### PR DESCRIPTION
**Pull request by bogdaniel**
While testing this bundle i was getting hit by an erorr that i can't find the babel preset es2017 which i solved it by installing.
I'm new to this webpack thing and to work like this with javascript. Some people might be in my position and try for hours to solve this. :D

### CODE REVIEW

File: frontend/encore/babel.rst [hunk 1]
```diff
@@ -7,6 +7,10 @@ Configuring Babel
 
 Need to extend the Babel configuration further? The easiest way is via
 ``configureBabel()``:
+But first let's install the babel preset.
+
+``yarn add babel-preset-es2017``
+This will install the preset that we will use later to configure the babel.
```
Comment (weaverryan) [inline frontend/encore/babel.rst]: I think this is indeed a good clarification! But what about adding this information as comments in the code block below (right above the es2017 line. We can say something like:

// make sure to install any presets you want to use
// yarn add ...

**Review by weaverryan — CHANGES_REQUESTED** [review]
Small comment, but I like it!

**Comment by bogdaniel** [comment]
I hope it is ok now :).

**Review by javiereguiluz — APPROVED** [review]
I propose a minor reword to make the comment easier to understand. But if you don't agree with it, please revert. Thanks!

**Comment by bogdaniel** [comment]
@javiereguiluz yeap it's ok =D

**Comment by xabbuh** [comment]
Thank you @bogdaniel.

**Pull request merged**